Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -602,15 +632,15 @@ private class TopSitesDataSource: NSObject, UICollectionViewDataSource {
             return nil
         }
 
-        if index >= self.historySites.count {
-            return self.suggestedSites[index - self.historySites.count]
-        }
-
         return self.historySites[index] as Site?
     }
 
     private func count() -> Int {
-        return historySites.count + suggestedSites.count
+        return historySites.count
```

Comment:
Since `historySites` is really a combination of history sites and suggested sites, we should probably rename this. Maybe just `sites`?